apis/sample/v1alpha1/groupversion_info.go

```diff
@@ -36,15 +38,3 @@ var (
 	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
 	SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
 )
-
-// MyType type metadata.
```

Comment:
@muvaf are we keeping old structure or updating? I was slightly confused by yours and @negz comments.